README.rst

```diff
@@ -1,61 +1,95 @@
 
-Component Usage::
+Environment Variables
+=====================
 
-   import Footer, { messages } from '@edx/frontend-component-footer';
+This component requires that the following environment variable be set by the consuming micro-frontend.
 
-   ...
+* ``LOGO_TRADEMARK_URL`` - This is a URL to a logo with trademark for use in the footer.
 
-   <Footer
-      onLanguageSelected={(languageCode) => {/* set language */}}
-      supportedLanguages={[
-         { label: 'English', value: 'en'},
-         { label: 'Español', value: 'es' },
-      ]}
-   />
+************
+Installation
+************
 
-- onLanguageSelected (optional)
-- supportedLanguages (optional)
+To install this footer into your Open edX micro-frontend, run the following command in your MFE:
 
-Styles (project.scss)::
+``npm i --save @edx/frontend-component-footer``
 
-   @import '@edx/frontend-component-footer/src/footer.scss';
+This will make the component available to be imported into your application.
 
+*****
+Usage
+*****
 
-Requirements
-------------
+This library has the following exports:
+
+* ``(default)``: The footer as a React component.
+* ``messages``: Internationalization messages suitable for use with `@edx/frontend-platform/i18n <https://edx.github.io/frontend-platform/module-Internationalization.html>`_
+* ``dist/footer.scss``: A SASS file which contains style information for the component.  It should be imported into the micro-frontend's own SCSS file.
+
+<Footer /> component props
+==========================
+
+* onLanguageSelected: Provides the footer with an event handler for when the user selects a
+  language from its dropdown.
+* supportedLanguages: An array of objects representing available languages.  See example below for object shape.
+
+Examples
+========
 
-This component uses ``@edx/frontend-i18n``. Any containing app must provide ``@edx/frontend-i18n`` as a peer dependency, and be wrapped inside an ``IntlProvider`` element, whether or not your consuming application is actually localized. For a basic default locale (English) version, follow the ``IntlProvider`` example in the sample application in `src/index.jsx <src/index.jsx>`__.
+Component Usage Example::
 
+  import Footer, { messages } from '@edx/frontend-component-footer';
+
+  ...
+
+  <Footer
+    onLanguageSelected={(languageCode) => {/* set language */}}
+    supportedLanguages={[
+      { label: 'English', value: 'en'},
+      { label: 'Español', value: 'es' },
+    ]}
+  />
+
+* `An example of minimal component and messages usage. <https://github.com/edx/frontend-template-application/blob/master/src/index.jsx#L23>`_
+* `An example of SCSS file usage. <https://github.com/edx/frontend-template-application/blob/master/src/index.scss#L9>`_
```

Comment:
here too have the concern about linking to master

Reply:
Fixed!